gatling-core/src/main/scala/io/gatling/core/check/ConditionalCheck.scala

```diff
@@ -0,0 +1,82 @@
+package io.gatling.core.check
+
+import com.typesafe.scalalogging.LazyLogging
+
+import io.gatling.core.session._
+import io.gatling.commons.validation.Validation
+import io.gatling.commons.validation.Success
+import io.gatling.core.check.extractor.Extractor
+import java.util.UUID
+import io.gatling.core.check.CheckResult
+import io.gatling.core.check.Check
+import io.gatling.core.check.DefaultFindCheckBuilder
+import io.gatling.core.check.{ Check, Extender, Preparer }
+import io.gatling.core.Predef._
```

Comment:
On this, my sbt on windows is anoying, it reformat all the sources, replacing /n by /r/n, ... Can I disable the default format and format only some files like go compiler do ?